package.json

```diff
@@ -42,24 +42,25 @@
     "react": "^0.14.0-alpha"
   },
   "devDependencies": {
-    "babel": "5.6.14",
-    "babel-core": "5.6.15",
-    "babel-eslint": "^3.1.20",
+    "babel": "^5.8.21",
+    "babel-core": "^5.8.22",
+    "babel-eslint": "^4.0.10",
     "babel-loader": "^5.3.0",
     "babel-plugin-object-assign": "^1.2.1",
     "chai": "^3.2.0",
-    "css-loader": "^0.15.6",
+    "component-playground": "github:jquense/component-playground",
+    "css-loader": "^0.16.0",
     "es5-shim": "^4.1.7",
-    "eslint": "^0.24.0",
-    "eslint-plugin-babel": "^1.0.0",
-    "eslint-plugin-mocha": "^0.4.0",
+    "eslint": "^1.2.0",
```

Comment:
This actually pretty much disables all the `eslint` rules, because of the changed defaults between 0.24 and 1.0.